backend/src/schemas/slide.schema.ts

```diff
@@ -1,82 +1,7 @@
 import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { Board } from './board.schema';
+import { SlideObject } from './slide-object.schema';
 
-export class SlideObject {
-  type: string;
-  version: string;
-  originX: string;
-  originY: string;
-  left: number;
-  top: number;
-  width: number;
-  height: number;
-  fill: string | null;
-  stroke: string | null;
-  strokeWidth: number;
-  strokeDashArray: number[] | null;
-  strokeLineCap: 'butt' | 'round' | 'square';
-  strokeDashOffset: number;
-  strokeLineJoin: 'miter' | 'round' | 'bevel';
-  strokeUniform: boolean;
-  strokeMiterLimit: number;
-  scaleX: number;
-  scaleY: number;
-  angle: number;
-  flipX: boolean;
-  flipY: boolean;
-  opacity: number;
-  shadow: {
-    color: string;
-    blur: number;
-    offsetX: number;
-    offsetY: number;
-  } | null;
-  visible: boolean;
-  backgroundColor: string;
-  fillRule: string;
-  paintFirst: 'fill' | 'stroke';
-  globalCompositeOperation: string;
-  skewX: number;
-  skewY: number;
-  path: Array<[string, number, number]>;
-  x1?: number;
-  x2?: number;
-  y1?: number;
-  y2?: number;
-  rx?: number;
-  ry?: number;
-  radius?: number;
-  startAngle?: number;
-  endAngle?: number;
-  fontFamily?: string;
-  fontWeight?: string;
-  fontSize?: number;
-  text?: string;
-  underline?: boolean;
-  overline?: boolean;
-  linethrough?: boolean;
-  textAlign?: 'left' | 'center' | 'right';
-  fontStyle?: 'normal' | 'italic';
-  lineHeight?: number;
-  textBackgroundColor?: string;
-  charSpacing?: number;
-  styles?: {
-    [key: string]: {
-      textDecoration: string;
-    };
-  }[];
-  direction?: 'ltr' | 'rtl';
-  pathStartOffset?: number;
-  pathSide?: 'left' | 'right' | 'center';
-  pathAlign?: 'baseline' | 'middle' | 'top' | 'bottom';
-  src?: string;
-  cropX?: number;
-  cropY?: number;
-  filters?: {
-    type: string;
-    value: number;
-  }[];
-}
 
 @Schema({ timestamps: true })
 export class Slide {
```

Comment:
The are no real relationships between [Slide & SlideObject](https://github.com/OBELISK-TEAM/OBELISK/blob/13c89eaf6763740340942541c2a997102846b473/backend/src/schemas/slide.schema.ts#L15) (one to many) and [Slide & Board](https://github.com/OBELISK-TEAM/OBELISK/blob/13c89eaf6763740340942541c2a997102846b473/backend/src/schemas/slide.schema.ts#L25) (many to one). Take a look at other relationships like this one between [User & Board](https://github.com/OBELISK-TEAM/OBELISK/blob/13c89eaf6763740340942541c2a997102846b473/backend/src/schemas/user.schema.ts#L45) and [Board & User](https://github.com/OBELISK-TEAM/OBELISK/blob/13c89eaf6763740340942541c2a997102846b473/backend/src/schemas/board.schema.ts#L50)